code/__HELPERS/unsorted.dm

```diff
@@ -1399,14 +1399,12 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 		/obj/item/food/grown,
 		/obj/item/food/grown/mushroom,
 		/obj/item/food/deepfryholder,
-		/obj/item/reagent_containers/food/snacks/clothing,
-		/obj/item/food/grown/shell, //base types
```

Comment:
u should move egg-plants to be subtypes of eggplants or grown if you do this, shell subtype doesnt do anything anyway

Reply:
what?

Reply:
u know just revert the shell thing from this